.testing/Makefile

```diff
@@ -88,6 +88,13 @@ SOURCE = $(wildcard $(BASE)/src/*/*.F90 $(BASE)/src/*/*/*.F90 $(BASE)/config_src
 all: $(foreach b,$(BUILDS),$(BUILD)/$(b)/MOM6)
 build.regressions: $(foreach b,symmetric target,$(BUILD)/$(b)/MOM6)
 
+# Shortcuts to build executables
+build-symmetric: $(BUILD)/symmetric/MOM6
+build-asymmetric: $(BUILD)/asymmetric/MOM6
+build-openmp: $(BUILD)/openmp/MOM6
+build-repro: $(BUILD)/repro/MOM6
+build-fms: $(FMS)/lib/libfms.a
+
```

Comment:
build.symmetric, build.asymmetric, etc to match the other rules?

Reply:
Once you get rid of the paths wouldn't the rule b build/symmetric ?

Reply:
build/symmetric/MOM6 but yeah basically